.github/workflows/build.yml

```diff
@@ -13,6 +13,7 @@ jobs:
 
     name: build-prod
     env:
+      ALBY_API_URL: "https://getalby.com/api"
```

Comment:
Do we still need WALLET_ROOT_URL, plus inconsistency with final "/"

Reply:
it is only used in the previous walletCreate method to create alby wallet, we can remove it!